### PR TITLE
update-mui-design-system

### PR DESCRIPTION
**Pull request by ficast**
image

### CODE REVIEW

File: packages/design-system-mui/components/ButtonWithLoading/styled.tsx [hunk 1]
```diff
@@ -1,6 +1,18 @@
...
+  ButtonProps,
+  CircularProgress as MuiCircularProgress,
+  CircularProgressProps,
+} from '@mui/material'
+import { FC } from 'react'
 
-export const Button = styled(MUIButton)(({ theme }) => ({
-  background: theme.palette.primary.main,
-}))
+export const Button = styled(MUIButton)(({ theme }) => ({})) as unknown as FC<ButtonProps>
```
Comment (anicioalexandre) [inline packages/design-system-mui/components/ButtonWithLoading/styled.tsx]: no really need to define it here if we are not customizing it

File: packages/design-system-mui/components/ButtonWithLoading/styled.tsx [hunk 2]
```diff
@@ -1,6 +1,18 @@
...
+
+export const CircularProgress = styled(MuiCircularProgress)(({ theme }) => ({
+  position: 'absolute',
```
Comment (anicioalexandre) [inline packages/design-system-mui/components/ButtonWithLoading/styled.tsx:6]: wouldn't be better if we show the loading state besides the children text? Not sure how the design looks like but seems better than overlapping the content

File: packages/design-system-mui/package.json [hunk 3]
```diff
@@ -21,6 +21,7 @@
   "dependencies": {
     "@emotion/react": "^11.10.0",
     "@emotion/styled": "^11.10.0",
+    "@mui/icons-material": "^5.8.4",
```
Comment (anicioalexandre) [inline packages/design-system-mui/package.json:24]: let's not add this until we need

Comment (ficast) [reply packages/design-system-mui/package.json:24]: I am using that

File: packages/design-system-mui/components/ButtonWithLoading/styled.tsx (hunk 2 above)
Comment (ficast) [reply packages/design-system-mui/components/ButtonWithLoading/styled.tsx:6]: you can always have your own loadingChildren to overwrite

**Review by anicioalexandre — APPROVED** [review]

**Pull request merged**